.github/workflows/tacos_unlock.yml

```diff
@@ -70,13 +69,11 @@ jobs:
       matrix:
         tf-root-module:
           ${{ fromJSON(needs.determine-tf-root-modules.outputs.slices) }}
-
     runs-on: ubuntu-latest
     permissions:
       contents: read
       pull-requests: write
       id-token: write
-
     env:
       TF_ROOT_MODULE: ${{matrix.tf-root-module}}
```

Comment:
this is still needed because the setup step breaks otherwise. 

https://github.com/getsentry/tacos-gha.demo/actions/runs/8425696855/job/23079804120

if we really wanted to remove this, it would require also editing/reverting `./tacos-gha/.github/actions/setup` which every other workflow is using, so just leaving it like this is better.

Reply:
No I think leaving things in this state is causing the system to get into an inconsistent state.